### PR TITLE
Ref/feature/panel size

### PR DESCRIPTION
**Pull request by TheSintoras**
Panel Size is now a setting instead of saving into its own file.

### CODE REVIEW

File: Assets/Scripts/System/WorkspaceSaveSystem.cs [hunk 1]
```diff
@@ -2,31 +2,41 @@
 using DynamicPanels;
 using System;
 using System.IO;
+using Zenject;
+using static Assets.Scripts.System.SettingsSystem;
```
Comment (cblech) [inline Assets/Scripts/System/WorkspaceSaveSystem.cs]: `using System.IO;` and `using static Assets.Scripts.System.SettingsSystem;` are not needed

File: Assets/Scripts/System/WorkspaceSaveSystem.cs [hunk 2]
```diff
@@ -2,31 +2,41 @@
...
 
 namespace Assets.Scripts.System
 {
+
```
Comment (cblech) [inline Assets/Scripts/System/WorkspaceSaveSystem.cs]: Code style: Don't put an empty line after opening a code block, i.e. after an `{`

File: UserSettings/EditorUserSettings.asset [hunk 3]
```diff
@@ -6,31 +6,31 @@ EditorUserSettings:
   serializedVersion: 4
   m_ConfigSettings:
     RecentlyUsedScenePath-0:
-      value: 22424703114646680e0b0227036c6c111b07142f1f2b233e2867083debf42d
+      value: 224247031146466e0307183b370f6b365932002b21382a353e662e30ebc63d10e7eb3becf30e3a2f2d31ec390a3b4f2afc031d12
```
Comment (cblech) [inline UserSettings/EditorUserSettings.asset]: Please reset changes to this file. It's not needed in this pull request

File: Assets/Scripts/System/WorkspaceSaveSystem.cs [hunk 4]
```diff
@@ -2,31 +2,41 @@
...
             byte[] data = PanelSerialization.SerializeCanvasToArray(canvas);
             string dataString = Convert.ToBase64String(data);
-            File.WriteAllText(saveFilePath, dataString);
+            settingSystem.panelSize.Set(dataString);
         }
 
         public void Load(DynamicPanelsCanvas canvas)
         {
-            if(!File.Exists(saveFilePath))
+            if(settingSystem.panelSize.Get()=="")
```
Comment (cblech) [inline Assets/Scripts/System/WorkspaceSaveSystem.cs]: Code style: please add some spaces between "commands". 
`if (settingSystem.panelSize.Get() == "")`
Hint: use auto formatting to fix these simple code style errors before committing

**Review by cblech — CHANGES_REQUESTED** [review]

File: UserSettings/EditorUserSettings.asset (hunk 3 above)
Comment (TheSintoras) [reply UserSettings/EditorUserSettings.asset]: Done

**Comment by MGH-Automator** [comment]
Task linked: [CU-3tmdjjp Save panel size in Project settings](https://app.clickup.com/t/3tmdjjp)

**Pull request merged**